src/xcomponentAPI.ts

```diff
@@ -1,35 +1,63 @@
 import { Connection, DefaultConnection } from "./communication/xcConnection";
 import { Session } from "./communication/xcSession";
 import { CompositionModel } from "./communication/xcomponentMessages";
+import { SSL_OP_ALLOW_UNSAFE_LEGACY_RENEGOTIATION } from "constants";
 let log = require("loglevel");
 
 class XComponentAPI {
     private connection: Connection;
 
     constructor() {
+        this.setLogLevel(log.levels.INFO);
```

Comment:
why do we set a log level in the constructor?
will that override user's defined loglevel?

Reply:
We just have to give it an initial value. It won't override user's defined loglevel

Reply:
Ok